lab_2_tokenize_by_bpe/main.py

```diff
@@ -69,6 +133,16 @@ def get_vocabulary(
     :param unknown_token: a token to signify an unknown token
     :return: dictionary in the form of <token: identifier>
     """
+    if not isinstance(word_frequencies, dict) or not isinstance(unknown_token, str):
+        return None
+    unique_tokens = set()
+    for word in word_frequencies.keys():
+        unique_tokens.update(word)
```

Comment:
this way you are adding all elements from the word, aren't you also supposed to add all symbols?

Reply:
solved